Check if scheduled mode is set when realtime is not available (#1474)
wazuh/wazuh-qa: docs/tests/integration/test_fim/test_files/test_basic_usage/test_basic_usage_realtime_unsupported.md

# Test basic usage realtime unsupported

> The test check for the correct transition to FIM mode scheduled when realtime is not avaible in the current OS and is
> set in the ossec.conf as a monitorization mode.

Reviewer: ```suggestion
The test checks for FIM to properly change from realtime to scheduled mode when it is not supported.
```

Author: Done in [commit](https://github.com/wazuh/wazuh-qa/pull/1474/commits/a46ae8a444c0c0a7f68436a4ccee7af9780437c6)

## General info

| Tier | Platforms | Time spent | Test file |
|:--:|:--:|:--:|:--:|
| 0 | Macos | 65 | test_basic_usage_realtime_unsupported.py

> | 0 | Solaris | 64 | test_basic_usage_realtime_unsupported.py

Reviewer: I believe the time spent column should be in `hh:mm:ss` format.

Author: From other reviews  I was told to use ss format only
